trigger/payloads.go

```diff
@@ -27,6 +27,7 @@ const (
 	evergreenSuccessColor    = "#4ead4a"
 	evergreenFailColor       = "#ce3c3e"
 	evergreenSystemFailColor = "#ce3c3e"
+	evergreenRunningColor    = "#FEF7E3"
```

Comment:
Not sure if you've tested this color but it seems a little dull. It looks like we use this in the UI (as an option) https://github.com/ablack12/evergreen/blob/6e1a16313ed11e8cce2934bcc2402b161c458c8e/public/static/less/global_styles.less#L66
This 
![image](https://user-images.githubusercontent.com/26798134/133294619-0406840a-5815-4036-afd6-116746c07010.png)
versus 
![image](https://user-images.githubusercontent.com/26798134/133294686-000b05ec-065b-4562-bbe2-e2065891daad.png)

Reply:
![image](https://user-images.githubusercontent.com/26491602/133314471-6f82f134-2759-4ebe-9f20-36af4e7d7046.png)